converter/biostudies.py

```diff
@@ -0,0 +1,176 @@
+from json_converter.json_mapper import JsonMapper
+from json_converter.post_process import default_to
+
+PROJECT_SPEC_BASE = [
+    {
+        'name': ['', default_to, 'Project Core - Project Short Name'],
```

Comment:
I see many of those arrays here:

```python
['', default_to, 'some_string']
```

Have you considered creating a factory function for this?
For example:

``` python
def make_default_to_spec(name):
   return ['', default_to, name]
```

You could even extend this to the entire name-value pair.

Reply:
@amnonkhen I get want you want, but I am not sure the specification would be more readable with that change. What do you think?